spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/amqp/RabbitAutoConfigurationTests.java

```diff
@@ -66,7 +72,6 @@
  * @author Greg Turnquist
  * @author Stephane Nicoll
  * @author Gary Russell
- * @author Stephane Nicoll
```

Comment:
Why are you doing this to me? 😭

Reply:
Looks like you are already listed 2 lines up. Double Stephane 😄

Reply:
Oh 😀

Reply:
I've removed my clone in 6f7501f